staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -472,6 +472,14 @@ func determineResyncPeriod(desired, check time.Duration) time.Duration {
 	return desired
 }
 
+func checkResyncPeriod(resyncPeriod time.Duration) time.Duration {
```

Comment:
Rename to validateResyncPeriod, to avoid confusion with the existing resyncCheckPeriod.